amen/audio.py

```diff
@@ -18,4 +20,43 @@ def __init__(self, file_path, convert_to_mono=False, sample_rate=22050):
         self.raw_samples = y
         self.num_channels = y.ndim
         self.duration = librosa.get_duration(y=y, sr=sr)
+        self.features = self.create_features()
 
+    def create_features(self):
+        """
+        Creates the various features in the features dict.
+        """
+        features = {}
+        features['centroid'] = self.get_centroid()
+        features['amplitude'] = self.get_amplitude()
+        return features
+
+    def get_centroid(self):
+        """
+        Gets spectral centroid data from librosa and loads it into a feature.
+        """
+        mono_samples = librosa.to_mono(self.raw_samples)
+        centroids = librosa.feature.spectral_centroid(mono_samples)
+        data = self._convert_to_dataframe(centroids, ['spectral_centroid'])
+        return data
+
+    def get_amplitude(self):
+        """
+        Gets amplitude data from librosa and loads it into a feature.
+        """
+        mono_samples = librosa.to_mono(self.raw_samples)
+        amplitudes = librosa.feature.rmse(mono_samples)
+        data = self._convert_to_dataframe(amplitudes, ['amplitude'])
+
+        return data
+
+    def _convert_to_dataframe(self, feature_data, columns):
+        """
+        Take feature data, convert to a pandas dataframe.
+        """
+        feature_data = feature_data[0]
```

Comment:
this is a little strange-looking.  is it necessary here?

also, can we docstring the parameters/returns of these functions in numpydoc style?